jekyll/_cci2/server/installation/phase-3-execution-environments.adoc

```diff
@@ -0,0 +1,949 @@
++
+[source,shell]
+----
+aws eks describe-cluster --name=<YOUR_CLUSTER_NAME>
+----
++
+This returns something similar to the following:
++
+[source,json]
+----
+{...
+"resourcesVpcConfig": {
+    "subnetIds": [
+        "subnet-033a9fb4be69",
+        "subnet-04e89f9eef89",
+        "subnet-02907d9f35dd",
+        "subnet-0fbc63006c5f",
+        "subnet-0d683b6f6ba8",
+        "subnet-079d0ca04301"
+    ],
+    "clusterSecurityGroupId": "sg-022c1b544e574",
+    "vpcId": "vpc-02fdfff4c",
+    "endpointPublicAccess": true,
+    "endpointPrivateAccess": false
+    }
+...
+"kubernetesNetworkConfig": {
+            "serviceIpv4Cidr": "10.100.0.0/16"
+        },
+...
+}
+----
++
+Then, using the VPCID you just found, run the following command to get the CIDR Block for your cluster. For AWS, the DNS Server is the third IP in your CIDR block (`CidrBlock`), for example your CIDR block might be `10.100.0.0/16`, so the third IP would be `10.100.0.2`.
++
+[source,shell]
+----
+aws ec2 describe-vpcs --filters Name=vpc-id,Values=<YOUR_VPCID>
+----
++
+This returns something like the following:
++
+[source,json]
+----
+{...
+"CidrBlock": "192.168.0.0/16",
+"DhcpOptionsId": "dopt-9cff",
+"State": "available",
+"VpcId": "vpc-02fdfff4c"
+...}
+----
+
+
+Once you have filled in the appropriate information, you can deploy your Nomad clients by running the following command:
+
+[source,shell]
+----
+terraform init
+----
+
+[source,shell]
+----
+terraform plan
+----
+
+[source,shell]
+----
+terraform apply
+----
+
+After Terraform is done spinning up the Nomad client(s), it outputs the certificates and keys needed for configuring the Nomad control plane in CircleCI server. Copy them somewhere safe. The apply process usually only takes a minute.
+
+// Stop hiding from GCP PDF:
+
+endif::env-gcp[]
+// Don't include this section in the AWS PDF:
+
+ifndef::env-aws[]
+
+[#gcp]
+==== GCP
+
+You need the following information:
+
+* The Domain name of the CircleCI Application
+* The GCP Project you want to run Nomad clients in.
+* The GCP Zone you want to run Nomad clients in.
+* The GCP Region you want to run Nomad clients in.
+* The GCP Network you want to run Nomad clients in.
+* The GCP Subnetwork you want to run Nomad clients in.
+
+A full example, as well as a full list of variables, can be found link:https://github.com/CircleCI-Public/server-terraform/tree/main/nomad-aws[here].
+
+Once you have filled in the appropriate information, you can deploy your Nomad clients by running the following commands:
+
+[source,shell]
+----
+terraform init
+----
+
+[source,shell]
+----
+terraform plan
+----
+
+[source,shell]
+----
+terraform apply
+----
+
+After Terraform is done spinning up the Nomad client(s), it outputs the certificates and key needed for configuring the Nomad control plane in CircleCI server. Copy them somewhere safe.
+
+endif::env-aws[]
+
+[#nomad-autoscaler]
+=== Nomad Autoscaler
```

Comment:
@circleci/server We have config for autoscaler split across both the Nomad Clients and Nomad Server section. Would it be ok for me take out these in their own autoscaling section, I think it's a bit confusing having them split across the sections. Just wondering if there are reasons not to do this.